.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# How to contribute
+Thank you for showing interest in contributing to Pretendo Network! As an open source project, Pretendo Network welcomes contributions from developers outside of our core team. We are a large project, made up of over 100 different repositories spanning various languages and scopes. For the sake of consistency and maintainability, we have created these guidelines to outline project goals, standards, and best practices. This document will cover both code contributions as well as going over how to report any bugs or issues with a service or documentation. By adhering to these guidelines, contributors can ensure that their contributions align with the project's vision and meet the quality standards expected by the community.
+
+### Contents
+- [Prerequisites](#prerequisites)
+- [Making Issues](#making-issues)
+- [Code Changes](#code-changes)
+	- [Consistency](#consistency)
+	- [Quality and Substance](#quality-and-substance)
+- [Commits](#commits)
+	- [Scale and Scope](#scale-and-scope)
+	- [Messages](#messages)
+- [Pull Requests](#pull-requests)
+
+# Prerequisites
+Before diving into the contribution process, we recommend familiarizing yourself with our project's existing repositories. By studying our existing codebases, issues, and pull requests you will get a better sense of direction of how we operate. Additionally, please review our [Code of Conduct](CODE_OF_CONDUCT.md) to ensure a respectful and inclusive environment for all participants.
+
+Given that our project's main goal is to emulate Nintendo Network, it is also highly recommended that you familiarize yourself with how these services operate. Information on these topics, and our libraries, can be found both on our [wiki](https://nintendo-wiki.pretendo.network/docs/) and our (in progress) [developer documentation](https://developer.pretendo.network/home).
+
+> [!NOTE]
+> Developer documentation is highly work in progress. Do not expect to find everything there at this time. Please refer to the wiki first and foremost.
+
+Finally, we recommend joining our [Discord server](https://invite.gg/pretendo) to see any roadmap plans or ask specific questions from our developers.
```

Comment:
Joining the discord is going to expose you to #pretendo-general, and it's really not a good way to reach out to us due to the flooding issues.

Reply:
I'm open to alternatives if you have them. I couldn't think of any other real way to do this. So far Discord has been the best way to contact me, at least, and some devs have (devs wanting to use our servers for their projects or our tools). Genuinely asking, do you have a better idea?

Reply:
It could be made a bit clearer that it is completely okay to ping a developer for genuine technical questions about the Pretendo servers, although it would be good to keep that in support threads instead of general. I didn't understand that until I joined the supporter chat and saw how willing you all were to answer questions because it's hard to see any of that with how fast general chat moves to other topics (even before the NN closure joins).

Reply:
That sounds fair 👍 if you have any specific suggestions for how to word this feel free to make a commit suggestion here

Reply:
I have had some luck redirecting technical questions into #support and having them still ping a dev, since it's at least easier to hold a conversation in its own thread. It's still not perfect because it won't get seen there unless OP finds the right person to ping, but it's better..